src/components/SearchResultListing/index.js

```diff
@@ -107,12 +132,14 @@ class SearchResultListing extends PureComponent {
 
             {instances.map((instance, index) =>
               (<tr key={`tr-${JSON.stringify(instance)}`}>
-                <td>
-                  <Checkbox
-                    checked={!!~selectedInstances.indexOf(instance)}
-                    onChange={this.handleToggleSelected(instance)}
-                  />
-                </td>
+                {
```

Comment:
Make closing HTML/JSX tag right below corresponding opening tag.  All over the file.  Like
```javascript
... => (
  <td>
    ...
  </td>
)

Reply:
Done.